datalab/mlalpha/_cloud_models.py

```diff
@@ -152,7 +143,10 @@ def _retrieve_versions(self, page_token, page_size):
     page_token = list_info.get('nextPageToken', None)
     return versions, page_token
 
-  def __iter__(self):
+  def get_iterator(self):
+    """Get iterator of versions so it can be used as
+       "for v in ModelVersions(model_name).get_iterator()".
+    """
     return iter(datalab.utils.Iterator(self._retrieve_versions))
 
   def get_version_details(self, version_name):
```

Comment:
It does. "return self._api.projects().models().versions().get(name=name).execute()"? It returns what the REST API call returns.